db/stats.py

```diff
@@ -1,4 +1,5 @@
-from db import create_cursor, commit, cache
+import db
+import db.cache
```

Comment:
Maybe memcached wrapper should be out of `db` package as well? What do you think?